webpack.config.node.js

```diff
@@ -55,7 +55,7 @@ function getExternals() {
 
 const webpackConfig = {
 	devtool: 'source-map',
-	entry: 'index.js',
+	entry: path.join( __dirname, 'index.js' ),
```

Comment:
I think it might be better to leave this change in #14731 and merge it in first. With a PR this big, we should make sure it doesn't touch anything outside of `client/extensions/woocommerce` if we can.